src/wp-includes/rest-api.php

```diff
@@ -84,8 +90,8 @@ function register_rest_route( $namespace, $route, $args = array(), $override = f
 		$arg_group['args'] = array_merge( $common_args, $arg_group['args'] );
 	}
 
-	$full_route = '/' . trim( $namespace, '/' ) . '/' . trim( $route, '/' );
-	rest_get_server()->register_route( $namespace, $full_route, $args, $override );
+	$full_route = '/' . trim( $clean_namespace, '/' ) . '/' . trim( $route, '/' );
```

Comment:
I didn't understand why you trim twice the local variable `$clean_namespace`?  It is unnecessary.
<img width="579" alt="Test result" src="https://user-images.githubusercontent.com/65644559/82409443-6d5f4500-9a76-11ea-97d5-bef44ee46be5.png">

Reply:
Good point! I've adjusted this in dbc648a97e.